### PR TITLE
Update manual label file naming

### PR DESCRIPTION
**Pull request by PaulBautin**
Done:
- Update manual labeling file naming following ivadomed [convention](https://github.com/ivadomed/ivadomed/wiki/repositories#derivatives), original comment: https://github.com/neuropoly/spinalcordtoolbox/issues/2859#issuecomment-679207356

### CODE REVIEW

**Comment by PaulBautin** [comment]
@jcohenadad, could you review this PR? Manual labeling file naming has been updated to ivadomed convention.

File: manual_correction.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def check_files_exist(dict_files, path_data):
     for task, files in dict_files.items():
         if files is not None:
             for file in files:
-                fname = os.path.join(path_data, sg.bids.get_subject(file), sg.bids.get_contrast(file) + get_rescaling(file) , file)
```
Comment (jcohenadad) [inline manual_correction.py:193]: if `get_rescaling()` function is not used anymore you can remove it

Comment (PaulBautin) [reply manual_correction.py:193]: Indeed, i have removed it in https://github.com/sct-pipeline/csa-atrophy/pull/54/commits/efda58d476754356deb2ff082ff39d0ed14e41a4

Comment (PaulBautin) [reply manual_correction.py:193]: Should i remove correct_pmj_detection() also?

Comment (jcohenadad) [reply manual_correction.py:193]: yup, in another PR

**Pull request merged**